Update 2.x docs with changes from 2.8.x (#9776)
RasaHQ/rasa: docs/versioned_docs/version-2.8.x/components.mdx

      smaller than the `ambiguity_threshold`.

:::caution Changed in 2.8

> Prior this version, `FallbackClassifier` used to set the confidence to 1.0, now it follows the fallback `threshold` config value.

Reviewer: ```suggestion
`FallbackClassifier` used to set the confidence to 1.0 but now follows the fallback `threshold` config value.
```
